dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -0,0 +1,223 @@
+from collections import defaultdict
+from . import utils, BaseColumnProfiler
+import itertools
+
+class TextProfiler(object):
+    type = 'unstructured_text'
+
+    def __init__(self, name, options=None):
+        """
+        Initialization of Text Profiler.
+
+        :param name: Name of the data
+        :type name: String
+        :param options: Options for the Text Profiler
+        :type options: UnstructuredTextOptions
+        """
+        self.name = name
+        self.sample_size = 0
+        self.times = defaultdict(float)
+        self.vocab = set()
+        self.word_count = defaultdict(int)
+        self.metadata = dict()
+
+        # TODO: Add line length
+        #self.line_length = {'max': None, 'min': None,...} #numeric stats mixin?
+
+        # these stop words are from nltk
+        self._stop_words = {
+            'i', 'me', 'my', 'myself', 'we', 'our', 'ours', 'ourselves', 'you',
+            "you're", "you've", "you'll", "you'd", 'your', 'yours', 'yourself',
+            'yourselves', 'he', 'him', 'his', 'himself', 'she', "she's", 'her',
+            'hers', 'herself', 'it', "it's", 'its', 'itself', 'they', 'them',
+            'their', 'theirs', 'themselves', 'what', 'which', 'who', 'whom',
+            'this', 'that', "that'll", 'these', 'those', 'am', 'is', 'are',
+            'was', 'were', 'be', 'been', 'being', 'have', 'has', 'had',
+            'having', 'do', 'does', 'did', 'doing', 'a', 'an', 'the', 'and',
+            'but', 'if', 'or', 'because', 'as', 'until', 'while', 'of', 'at',
+            'by', 'for', 'with', 'about', 'against', 'between', 'into',
+            'through', 'during', 'before', 'after', 'above', 'below', 'to',
+            'from', 'up', 'down', 'in', 'out', 'on', 'off', 'over', 'under',
+            'again', 'further', 'then', 'once', 'here', 'there', 'when',
+            'where', 'why', 'how', 'all', 'any', 'both', 'each', 'few', 'more',
+            'most', 'other', 'some', 'such', 'no', 'nor', 'not', 'only', 'own',
+            'same', 'so', 'than', 'too', 'very', 's', 't', 'can', 'will',
+            'just', 'don', "don't", 'should', "should've", 'now', 'd', 'll',
+            'm', 'o', 're', 've', 'y', 'ain', 'aren', "aren't", 'couldn',
+            "couldn't", 'didn', "didn't", 'doesn', "doesn't", 'hadn', "hadn't",
+            'hasn', "hasn't", 'haven', "haven't", 'isn', "isn't", 'ma',
+            'mightn', "mightn't", 'mustn', "mustn't", 'needn', "needn't",
+            'shan', "shan't", 'shouldn', "shouldn't", 'wasn', "wasn't",
+            'weren', "weren't", 'won', "won't", 'wouldn', "wouldn't"
```

Comment:
NLTK is probably not the selection of words like this I'd use - reason:

1. Look at `shouldn't`, I typically type that `shouldnt`
2. Their lists are often less inclusive (as shown above)

I believe this is already in somewhere, but bringing up my concern for poor selection.

Reply:
Ah, good points, do you have a suggestion? I am unfamiliar with stop word datasets

Reply:
@lettergram my assumption is that NLTK would use this as a base, but also do a strip on the `'` to include in the set so they wouldn't repeat the words multiple times. maybe i'm wrong?

Reply:
imo, we should have a txt file saving all stopwords, which should be updated every time we want

Reply:
actually, another way is to allow stopwords as an input argument for the user

Reply:
agreed, however, we haven't built out the options for it yet, so this should work for now. I will create an issue to make the options class